mods/gen3/random-teams.js

```diff
@@ -385,38 +385,33 @@ class RandomGen3Teams extends RandomGen4Teams {
 			item = 'Stick';
 		} else if (template.species === 'Marowak') {
 			item = 'Thick Club';
-		} else if (template.species === 'Pikachu') {
-			item = 'Light Ball';
 		} else if (template.species === 'Shedinja') {
 			item = 'Lum Berry';
-		} else if (template.species === 'Unown') {
-			item = 'Twisted Spoon';
-		} else if (template.species === 'Wobbuffet') {
-			item = ['Leftovers', 'Sitrus Berry'][this.random(2)];
 		} else if (template.species === 'Slaking') {
 			item = 'Choice Band';
+		} else if (template.species === 'Unown') {
+			item = 'Twisted Spoon';
 		} else if (hasMove['trick']) {
 			item = 'Choice Band';
 		} else if (hasMove['bellydrum']) {
-			item = 'Sitrus Berry';
+			item = 'Salac Berry';
 		} else if (hasMove['rest'] && !hasMove['sleeptalk'] && ability !== 'Natural Cure' && ability !== 'Shed Skin') {
 			item = 'Chesto Berry';
 
 		// Medium priority
-		} else if (hasMove['endeavor'] || hasMove['flail'] || hasMove['reversal'] || hasMove['endure']) {
-			if (template.baseStats.spe < 108) {
+		} else if (hasMove['leechseed']) {
+			item = 'Leftovers';
+		} else if (hasMove['endeavor'] || hasMove['flail'] || hasMove['reversal'] || hasMove['endure'] ||
+			hasMove['substitute'] && counter.Status < 3 && template.baseStats.hp + template.baseStats.def + template.baseStats.spd < 250 && this.random(2)) {
+			if (template.baseStats.spe <= 90 && !counter['speedsetup']) {
 				item = 'Salac Berry';
-			} else if (counter.Physical > counter.Special) {
+			} else if (counter.Physical >= counter.Special) {
 				item = 'Liechi Berry';
 			} else {
 				item = 'Petaya Berry';
 			}
-		} else if (counter.Physical >= 4 && !hasMove['bodyslam'] && !hasMove['fakeout'] && !hasMove['rapidspin']) {
+		} else if ((counter.Physical >= 4 || counter.Physical >= 3 && counter.Special === 1 && this.random(2)) && !hasMove['bodyslam'] && !hasMove['fakeout'] && !hasMove['rapidspin']) {
 			item = 'Choice Band';
-		} else if (counter.Special >= 4 || (counter.Special >= 3 && hasMove['batonpass'])) {
-			item = template.baseStats.spe >= 60 && template.baseStats.spe <= 108 && ability !== 'Speed Boost' && !counter['priority'] && this.random(3) ? 'Salac Berry' : 'Petaya Berry';
-		} else if (hasMove['outrage'] && counter.setupType) {
-			item = 'Lum Berry';
```

Comment:
Is outrage + lum berry not that useful?

Reply:
Outrage only has 90 BP in ADV so it's a terrible move that's not in any Pokemon's randbat movepool.

Reply:
I could of sworn it was better in gen 3 / in use. But it seems your right.